packages/vue-test-workspace/vue-tsc/non-strict-template/#2157/main.vue

```diff
@@ -0,0 +1,11 @@
+<script setup lang="ts">
+declare module 'vue' {
+  export interface GlobalComponents {
+    Transition: typeof import('vue')['Transition'];
+  }
+};
```

Comment:
Even though it's within a specific file, it will affect all tests, I believe.
If so then maybe use some more unique name so that there is no chance of affecting other tests with that?